### ISSUE TITLE
[SUGGESTION] Switch input qualifiers from file to path

### ISSUE THREAD

**Issue by KrisDavie** [vib-singlecell-nf/vsn-pipelines#185]
**Is your feature request related to a problem? Please describe.**
In Nextflow >=  19.10.0, it is recommended to use the `path` qualifier rather than the `file` qualifier.
See: https://www.nextflow.io/docs/latest/process.html?#input-of-type-path

**Describe the solution you'd like**
Use the `path` qualifier rather than the `file` qualifier.

**Describe alternatives you've considered**
/

**Additional context**
Path enables some extra options, like staging files as specific names within processes

**Comment by dweemx** [comment]
Hey, I already closed an issue related to this https://github.com/vib-singlecell-nf/vsn-pipelines/issues/56
Are there some parts not using `path` ?

**Comment by KrisDavie** [comment]
Yeah, it seems that there are quite a lot of places. `ctrl-shift-f` in vscode and searching for `file(` brings up a lot of results. A lot are in cellRanger modules but there are a lot in other places too.

**Comment by dweemx** [comment]
I see; Cell Ranger and in Nemesh pipepline
However, careful beacause the `path` is a qualifier and not a constructor. hence, all the `file` should not be replaced by `path`. `path` is a qualifier and not a constructor.
